src/components/deprecation/deprecatedEthWallet.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import 'antd/dist/antd.dark.css'
+
+class DeprecatedEthWallet extends React.Component {
```

Comment:
What steps are needed to get this component to render so I can see what it looks like?

Reply:
Remove unnecessary components.

Reply:
You can't. Forgot I removed the header switch. I'll remove the component.

Reply:
2e23175